src/transformers/tokenization_utils_base.py

```diff
@@ -3274,11 +3274,23 @@ def prepare_seq2seq_batch(
             The full set of keys ``[input_ids, attention_mask, labels]``, will only be returned if tgt_texts is passed.
             Otherwise, input_ids, attention_mask will be the only keys.
         """
+        # docstyle-ignore
+        example = """
+
+model_inputs = tokenizer(src_texts, ...)
+with tokenizer.as_target_tokenizer():
+    labels = tokenizer(tgt_texts, ...)
+model_inputs["labels"] = labels["input_ids"]
+"""
```

Comment:
Can we put the whole warning in a string defined outside of this function? It'll be more readable this way.

Reply:
Totally agree.